src/Umbraco.Cms.Integrations.Script.Cookiebot/Views/Partials/UmbracoCms.Integrations/Scripts/Cookiebot/Cookiebot.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage
+
+@inject Microsoft.Extensions.Configuration.IConfiguration Configuration
+
+@using Umbraco.Cms.Integrations.Script.Cookiebot
+
+@{
+    var scriptSrc = "https://consent.cookiebot.com/" + Configuration[$"{Constants.SettingsPath}:Id"] + "/cd.js";
```

Comment:
Again just in the aim of really simplifying things, maybe we can just hard-code the configuration path here and remove the constant.  It's not used anywhere else.  And that way there's only the `.cshtml` file that needs creating, no need for any `.cs`.